### PR TITLE
Feature/statistic data service

### PR DESCRIPTION
**Pull request by mbarsott**
Implement the API to get official statistic data about countries and the world.

### CODE REVIEW

File: statistics-data-service/obj/StatisticsDataService.csproj.nuget.dgspec.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
+  "format": 1,
+  "restore": {
+    "C:\\code\\dev\\covid-stat\\statistics-data-service\\StatisticsDataService.csproj": {}
```
Comment (MadKimchi) [inline statistics-data-service/obj/StatisticsDataService.csproj.nuget.dgspec.json:4]: will this work cross platform and is this supposed to be included?

File: statistics-data-service/bin/Debug/netcoreapp3.1/StatisticsDataService.runtimeconfig.dev.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "runtimeOptions": {
+    "additionalProbingPaths": [
+      "C:\\Users\\fledg\\.dotnet\\store\\|arch|\\|tfm|",
```
Comment (MadKimchi) [inline statistics-data-service/bin/Debug/netcoreapp3.1/StatisticsDataService.runtimeconfig.dev.json:4]: .dev sounds like an environment file. Should this be included? and if it does, C:\\ would not work on cross platform running environment

File: statistics-data-service/obj/Debug/netcoreapp3.1/StatisticsDataService.csproj.FileListAbsolute.txt [hunk 3]
```diff
@@ -0,0 +1,43 @@
+C:\code\dev\covid-stat\statistic-data-service\bin\Debug\netcoreapp3.1\appsettings.Development.json
+C:\code\dev\covid-stat\statistic-data-service\bin\Debug\netcoreapp3.1\appsettings.json
+C:\code\dev\covid-stat\statistic-data-service\bin\Debug\netcoreapp3.1\Properties\launchSettings.json
+C:\code\dev\covid-stat\statistic-data-service\bin\Debug\netcoreapp3.1\StatisticsDataService.exe
```
Comment (MadKimchi) [inline statistics-data-service/obj/Debug/netcoreapp3.1/StatisticsDataService.csproj.FileListAbsolute.txt:4]: Same comments as above?

File: statistics-data-service/obj/StatisticsDataService.csproj.nuget.dgspec.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  },
+  "projects": {
+    "C:\\code\\dev\\covid-stat\\statistics-data-service\\StatisticsDataService.csproj": {
+      "version": "1.0.0",
+      "restore": {
+        "projectUniqueName": "C:\\code\\dev\\covid-stat\\statistics-data-service\\StatisticsDataService.csproj",
+        "projectName": "StatisticsDataService",
+        "projectPath": "C:\\code\\dev\\covid-stat\\statistics-data-service\\StatisticsDataService.csproj",
```
Comment (MadKimchi) [inline statistics-data-service/obj/StatisticsDataService.csproj.nuget.dgspec.json:12]: same comment as above

File: statistics-data-service/obj/StatisticsDataService.csproj.nuget.g.props [hunk 5]
```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition=" '$(ExcludeRestorePackageImports)' != 'true' ">
+    <RestoreSuccess Condition=" '$(RestoreSuccess)' == '' ">True</RestoreSuccess>
+    <RestoreTool Condition=" '$(RestoreTool)' == '' ">NuGet</RestoreTool>
+    <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">$(MSBuildThisFileDirectory)project.assets.json</ProjectAssetsFile>
+    <NuGetPackageRoot Condition=" '$(NuGetPackageRoot)' == '' ">$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
+    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">C:\Users\fledg\.nuget\packages\;C:\Program Files\dotnet\sdk\NuGetFallbackFolder</NuGetPackageFolders>
```
Comment (MadKimchi) [inline statistics-data-service/obj/StatisticsDataService.csproj.nuget.g.props:8]: same comment as above

File: statistics-data-service/obj/project.assets.json [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+      "projectName": "StatisticsDataService",
+      "projectPath": "C:\\code\\dev\\covid-stat\\statistics-data-service\\StatisticsDataService.csproj",
+      "packagesPath": "C:\\Users\\fledg\\.nuget\\packages\\",
+      "outputPath": "C:\\code\\dev\\covid-stat\\statistics-data-service\\obj\\",
+      "projectStyle": "PackageReference",
+      "fallbackFolders": [
+        "C:\\Program Files\\dotnet\\sdk\\NuGetFallbackFolder"
+      ],
+      "configFilePaths": [
+        "C:\\Users\\fledg\\AppData\\Roaming\\NuGet\\NuGet.Config",
```
Comment (MadKimchi) [inline statistics-data-service/obj/project.assets.json:140]: same

**Comment by mbarsott** [comment]
bin and obj folders where you commented are result of local compilation, they should not be included in the repo and should be ignored with the .gitignore I added to the project folder. I will check why it is not working.

**Comment by mbarsott** [comment]
Removed ide and generated files from source control

**Pull request merged**